src/services/OrderSearchParams.php

```diff
@@ -24,38 +24,33 @@ public function getParams()
             return Translations::$plugin->translator->translate($category, $status);
         }, Translations::$plugin->orderRepository->getOrderStatuses());
         
-        $query = Craft::$app->request->getParam('criteria') ? Craft::$app->request->getParam('criteria') : Craft::$app->request->getParam('');
-
-
-        $sourceSite = isset($query['sourceSite']) ? $query['sourceSite'] : null;
-        $targetSite = isset($query['targetSite']) ? $query['targetSite'] : null;
-        $targetSite = isset($query['targetSite']) ? $query['targetSite'] : null;
-        $startDate = isset($query['startDate']) ? $query['startDate'] : null;
-        $endDate = isset($query['endDate']) ? $query['endDate'] : null;
-        $status = isset($query['status']) ? $query['status'] : null;
-
-        $params = array();
-
-        if ($sourceSite && array_key_exists($sourceSite, $sites)) {
-            $params['sourceSite'] = $sourceSite;
-        }
-
-        if ($targetSite && array_key_exists($targetSite, $sites)) {
-            $params['targetSite'] = $targetSite;
-        }
-
-        if ($startDate && isset($startDate['date']) && preg_match('#^\d{1,2}/\d{1,2}/\d{4}$#', $startDate['date'])) {
-            $params['startDate'] = $startDate['date'];
-        }
-
-        if ($endDate && isset($endDate['date']) && preg_match('#^\d{1,2}/\d{1,2}/\d{4}$#', $endDate['date'])) {
-            $params['endDate'] = $endDate['date'];
-        }
-
-        if ($status && array_key_exists($status, $statuses)) {
-            $params['status'] = $status;
```

Comment:
@shnsumit same with this, I'm pretty sure it was used in the Craft 2 plugin for reporting, but can you please verify it's not used anywhere else in the plugin?